test/common/quic/active_quic_listener_test.cc

```diff
@@ -94,6 +94,10 @@ class ActiveQuicListenerPeer {
   }
 
   static bool enabled(ActiveQuicListener& listener) { return listener.enabled_->enabled(); }
+
+  static Network::Socket* socket(ActiveQuicListener& listener) {
```

Comment:
nit: Can this return a ref instead to make it clear it's non-null?

Reply:
done